File: Web/godofgeeks23_AviralSrivastava_2024cse1056_2/gettingStarted/fibonaccitilln.cpp
#include <iostream>
using namespace std;
int fibo(int n)
{
    if(n==1)
        return 0;
    if(n==2)
        return 1;
    else
        return (fibo(n-1)+fibo(n-2));
}
int main(int argc, char **argv)
{
    int n;
    cin >> n;
    
    //write your code here
    for(int i=1;i<=n;i++)
        cout<<fibo(i)<<endl;
}

File: AR-VR/Vidushi7555_VidushiPandey_2024ec1126_2/GettingStarted/FibonaccitillN.java
/******************************************************************************

                            Online Java Compiler.
                Code, Compile, Run and Debug java program online.
Write your code in this editor and press "Run" button to execute it.

*******************************************************************************/

import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner sc=new Scanner(System.in);
      int n=sc.nextInt();
      int i,a=0,b=1,c=0;
      if(n<0)
      return;
      System.out.println(a + " ");
      for(i=1;i<n;i++)
      {
        System.out.println(b + " ");
        c=a+b;
        a=b;
        b=c;
      }// write your code here
  }
}